dts/bindings/pwm/raspberrypi,pico-pwm.yaml

```diff
@@ -16,88 +16,72 @@ properties:
 
   divider-int-0:
     type: int
-    default: 1
     description: |
       The integral part of the divider for pwm slice 0.
       This number should be in the range 1 - 255. Defaults
```

Comment:
Does it still default to the reset value?

Reply:
Yes, it does. It is set in this part.

https://github.com/zephyrproject-rtos/zephyr/pull/78786/files?diff=unified&w=0#diff-eb856606ea8e48c50d21f12c8393f76f68686f686d66d8817262913e99aff87aR190-R192